dev-python/nautilus-python/nautilus-python-4.0.ebuild

```diff
@@ -28,21 +28,22 @@ RDEPEND="
 "
 DEPEND="${RDEPEND}"
 BDEPEND="
+	gtk-doc? ( dev-util/gtk-doc )
 	>=dev-util/gtk-doc-am-1.14
 	virtual/pkgconfig
 "
 
 src_configure() {
 	local emesonargs=(
-		$(meson_feature doc docs)
+		$(meson_feature gtk-doc docs)
 	)
 	meson_src_configure
 }
 
 src_install() {
 	meson_src_install
 
-	use doc && mv "${ED}/usr/share/doc/${PN}/"* "${ED}/usr/share/doc/${P}" || die
```

Comment:
When I tried to install with `-doc` it just called die

Reply:
Ah, probably because you don't have gtk-doc installed and I didn't add the dependency?

Reply:
I don't think so:

```sh
false && echo "done" || echo "not done"
true && echo "done" || echo "not done"
```

the first one will print "not done" (equivalent to `die` here)

Reply:
Ah, thanks.